runtime/lua/vim/treesitter/dev.lua

```diff
@@ -42,22 +42,24 @@ local TSTreeView = {}
 ---@param lang string Language of the tree currently being traversed
 ---@param injections table<string, TSP.Injection> Mapping of node ids to root nodes
 ---                  of injected language trees (see explanation above)
+---@param field string|nil The field of the current node
 ---@param tree TSP.Node[] Output table containing a list of tables each representing a node in the tree
-local function traverse(node, depth, lang, injections, tree)
+local function traverse(node, depth, lang, injections, field, tree)
```

Comment:
I'm really sorry to be picky here, but it would be even better if we can move the parameter `field` before `lang` (e.g. next to `depth`), because:

- `node`, `depth`, and `field` are the parameters that represent the current context of the *node* being visited
- `injections` and `tree` are for the bigger context (*tree*)
- and this is a local function, so signature change is fine.